hadoop-hdds/docs/content/concept/_index.zh.md

```diff
@@ -2,7 +2,7 @@
 title: 概念
 date: "2017-10-10"
 menu: main
-weight: 6
+weight: 3
```

Comment:
Could you please explain the reason why `weight` is changed here? @flirmnave

Reply:
Hi,
I see that it weight 3 in english version.
https://github.com/apache/hadoop-ozone/blame/master/hadoop-hdds/docs/content/concept/_index.md#L5
So I change it, Should I change it back?

Reply:
No, it is the right thing to do, sorry for not noticing this.

Reply:
It’s ok.